docs/userguide/monitoring.rst

```diff
@@ -49,66 +54,68 @@ Here's an example configuration that logs monitoring information to a local sqli
 -------------
 
-Run the `parsl-visualize` utility::
+To view the web dashboard during or after a Parsl program has executed, you
+need to first run the `parsl-visualize` utility::
 
    $ parsl-visualize
 
-If your monitoring database is not the default of `monitoring.db` in the current working directory,
-you can specify a different database URI on the command line.  For example, if the full path
-to your `monitoring.db` is `/tmp/monitoring.db`, run::
+By default, this command expects that the default `monitoring.db` database is used
+in the current working directory. Other database can be loaded by passing
+the database URI on the command line.  For example, if the full path
+to the database is `/tmp/my_monitoring.db`, run::
 
-   $ parsl-visualize sqlite:////tmp/monitoring.db
+   $ parsl-visualize sqlite:////tmp/my_monitoring.db
 
-By default, the visualization web server listens on `127.0.0.1:8080`. If you are running on a machine with a web browser, you can access viz_server in the browser via `127.0.0.1:8080`. If you are running on the login node of a cluster, to access viz_server in a local machine's browser, you can use an ssh tunnel from your local machine to the cluster::
+By default, the visualization web server listens on `127.0.0.1:8080`. If the web server is deployed on a machine with a web browser, the dashboard can be accessed in the browser at `127.0.0.1:8080`. If the web server is deployed on a remote machine, such as the login node of a cluster, you will need to use an ssh tunnel from your local machine to the cluster::
 
    $ ssh -L 50000:127.0.0.1:8080 username@cluster_address
 
-This binds your local machine's port 50000 to the remote cluster's port 8080. This allows you to access viz_server directly on your local machine's browser via `127.0.0.1:50000`. 
+This command will bind your local machine's port 50000 to the remote cluster's port 8080.
+The dashboard can then be accessed via the local machine's browser at `127.0.0.1:50000`. 
 
-.. warning:: Below is an alternative to host the viz_server on a cluster, which may violate the cluster's security policy. Please check with your cluster admin before doing this.
-If the cluster allows you to host a web server on its public IP address with a specific port (i.e., open to Internet via `public_IP:55555`), you can run::
+.. warning:: Alternatively you can deploy the visualization server on a public interface. However, first check that this is allowed by the cluster's security policy. The following example shows how to deploy the web server on a public port (i.e., open to Internet via `public_IP:55555`)::
 
    $ parsl-visualize --listen 0.0.0.0 --port 55555
 
+
 Workflows Page
 ^^^^^^^^^^^^^^
 
-The workflows page lists all instances of a Parsl workflow that has been executed with monitoring turned on.
-It also gives a high level overview of workflow runs as a table as shown below:
+The workflows page lists all Parsl workflows that have been executed with monitoring enabled
```

Comment:
why is this workflows rather than programs?

Reply:
or maybe program executions?

Reply:
Leaving as workflows to match the monitoring UI. Will update when we update the UI.